src/pages/Home/components/PopupMarker/index.jsx

```diff
@@ -18,8 +18,9 @@ export default ({ lat, lng, business, history, setCurrentMarker, currentMarker }
             {currentMarker === business.id && (
                 <InfoBox
                     position={{
-                        lat: lat,
-                        lng: lng - 0.02222,
+                        lat,
+                        lng,
+                        // lng: lng - 0.02222, Causes problems especially zooming in and out while popup is open
```

Comment:
but the same problem for zooming in and out exists without  the `- 0.02222`. i want the the popup to appear under the marker. not to the right of it.
phones have more space vertically than horizontally.

Reply:
No, that problem is fixed when you don't play with the coordinates, but the popup remains to the bot-right of the selection.

If it is that necessary, the popup can be in the center by replacing InfoBox with InfoWindow, but creating a custom InfoWindow popup is very limited, as in our div will have to be inside the pre-made window.

![image](https://user-images.githubusercontent.com/55396239/77659126-54f41200-6f80-11ea-946b-aaaed2fcbb3a.png)

I can make it this way if it is to your satisfaction, Senpai Mario.

Reply:
there is a lot of dead space in that popup. remove the border and minimize the dead space

Reply:
I can't do a lot about this dead space, as you can see this is the area I can control:

![image](https://user-images.githubusercontent.com/55396239/77672147-31859300-6f91-11ea-9c44-605357045d7a.png)

Reply:
![Peek 2020-03-26 18-47](https://user-images.githubusercontent.com/24195641/77672999-457dc480-6f92-11ea-8c2d-4326d19e3895.gif)
okay then just fix this

Reply:
Strange... how did you trigger this bug?

Reply:
i just opened it chrome dev tools

Reply:
i tried refreshing but still the same

Reply:
Gave it a redesign, now it should work on all screens dynamically.

Reply:
again it's way too big
![Screenshot_2020-03-27-11-18-55](https://user-images.githubusercontent.com/24195641/77736320-1f513680-701d-11ea-872d-df332f5f763b.png)
this is on my phone, if you want to test the app on your mobile (i mena if you haven't figured it out already) change the API_URL to the heroku one and run the server, and then go to you laptop's port 3000 in your mobile